src/components/organisms/Nav.js

```diff
@@ -132,14 +183,42 @@ const Nav = () => {
             confirmText="Sign Up"
             cancelHandler={() => setModalState(modalStates.closed)}
           >
+            <label htmlFor="name">
+              Display Name
+              <input
+                type="text"
+                name="name"
+                value={usernameInput}
+                onChange={() => { handleNameChange(); }}
+                onBlur={() => { validation(); }}
```

Comment:
you're saying to run these functions without passing the event object, now. you want to make sure you pass the event object as both these functions are expecting that object via parameters. if this is confusing to you, just put it back to the way you had it and we can discuss it later.
```suggestion
                onChange={(event) => { handleNameChange(event); }}
                onBlur={(event) => { validation(event); }}
```